config/ignition_fortress_ubuntu_focal.yaml

```diff
@@ -0,0 +1,96 @@
+name: ignition_fortress_ubuntu_focal
+method: http://packages.osrfoundation.org/gazebo/ubuntu-stable
+suites: [focal]
```

Comment:
I think we're hoping for Fortress to be upstream in Ubuntu Jammy correct? We'll have these Fortress packages for Focal but over the next month Rolling development will jump over to the Jammy pre-releases and our upstreaming efforts are still underway. I can see some sense in adding Fortress to the bootstrap repo for Jammy in the interim and then removing those packages once they become available upstream. But I have to refresh myself on how hard it is to _remove_ packages from the bootstrap repository and ROS repositories once imported.

Reply:
Correct. One thing to keep in mind though is that relying on upstream will lock us into a fixed version, which wouldn't be great.

I know this has been brought up before, but I'm not sure we got to a consensus. I think ideally we'd have a way to continuously push updates to the Ignition packages consumed by ROS users.